libs/blocks/global-navigation/global-navigation.js

```diff
@@ -173,6 +173,12 @@ const getMessageEventListener = () => {
   };
 };
 
+const getSignInCtaStyle = () => {
```

Comment:
Nit: 
```suggestion
const getSignInCtaStyle = () =>
  (getMetadata('signin-cta-style') === 'primary' ||
   getConfig()?.unav?.profile?.signInCtaStyle === 'primary')
    ? 'primary'
    : 'secondary';
```

Reply:
@bandana147 Above suggestion have some linting issues.
Updated the suggestion a bit as mentioned below:
```suggestion
const getSignInCtaStyle = () => {
  const isPrimary = (
    getMetadata('signin-cta-style') === 'primary'
    || getConfig()?.unav?.profile?.signInCtaStyle === 'primary'
  );
  return isPrimary ? 'primary' : 'secondary';
};
```